src/accelerate/utils/launch.py

```diff
@@ -67,7 +67,7 @@ def _get_mpirun_args():
     mpirun_version = subprocess.check_output([mpi_app, "--version"])
 
     if b"Open MPI" in mpirun_version:
-        return mpi_app, "--hostfile", "-n", "--npernode"
+        return mpi_app, "--hostfile", "-n", "--npernode", "--bind-to"
```

Comment:
Adding the `--bind-to` argument from MPIRun is essential to ensuring multicpu runs with as many CPU threads as specified.